contracts/axelar-gateway/src/testutils.rs

```diff
@@ -31,19 +31,19 @@ pub fn initialize(
     previous_signers_retention: u32,
     num_signers: u32,
 ) -> TestSignerSet {
-    let signers = generate_signers_set(env, num_signers, BytesN::random(env));
-    let signers_set = vec![&env, signers.signers_set.clone()];
+    let signer_set = generate_signers_set(env, num_signers, BytesN::random(env));
+    let signers = vec![&env, signer_set.signers.clone()];
```

Comment:
confusing to use signers for a list and the individual weighted signers. Instead create a vector in the call below so you don't need to name it?